tuned/plugins/plugin_audio.py

```diff
@@ -96,12 +98,13 @@ def _get_timeout(self, device, ignore_missing=False):
 		return None
 
 	@command_set("reset_controller", per_device = True)
-	def _set_reset_controller(self, value, device, sim):
+	def _set_reset_controller(self, value, device, sim, no_error, remove):
```

Comment:
I assume the `no_error` parameter here is a mistake?

Reply:
Good catch, thanks. It was residue of my previous attempt to fix this problem :) It should be fixed now.